content/project-organization/quality-council/processes/issue-management.md

```diff
@@ -120,126 +135,132 @@ The handling of issues follows a detailed process. Here are the different steps,
 {{% /notice %}}
 
-#### Labelling
+### Labelling
 
-**This step is crucial.** Labelling consists on tagging the issue according to its different characteristics. This step is **critical** because it helps us find a bug when we’re looking for a duplicate. The different tags are:
+**This step is critical** cause **labelling** consists on tagging the issue according to its different characteristics and it helps us find a bug when we’re looking for a `duplicate`. The different tags are:
 
-- Type (bug or feature),
-- [Severity (major, minor, ...)][how-issues-are-sorted],
-- Regression or not,
-- Verified (if we reproduce the issue)
-- Category (FO/BO + affected page or concerned feature),
+- Type (`Bug` or `Task`),
+- [Severity (major, minor, ...)](../../../../get-involved/report-issues/how-issues-are-sorted/)
+- *Deprecated - Regression or not (since August 2023),*
+- `Verified` (if we reproduce the issue)
+- Category (`FO`/`BO`),
+- Affected page or concerned feature (ex: `Order`, `Module`, …),
 - Affected version(s),
 - Status (NMI, Needs Specs, Ready...)
 - ... (non-exhaustive list)
 
-A list of labels and their use is [available here](https://github.com/PrestaShop/PrestaShop/labels).
+A list of labels and their use is [available here](https://docs.google.com/spreadsheets/d/118iqWIkW24Rwn6eY8_7FnPhAacGXwA2RdGPzR_LXvy4/edit#gid=0).
 
-#### Connecting an issue to an EPIC
+### Connecting an issue to an EPIC
 
-When the issue is a **bug that was reproduced** (meaning with the `Verified` label), it MUST in most cases be connected to an `EPIC` if it already exists.
+{{% notice warning %}}
+When the issue is a **bug that was reproduced** (meaning with the `Verified` label), it MUST in most cases be connected to an Epic if it already exists.
+{{% /notice %}}
 
-To do this, perform a search using this filter [EPIC list](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aopen+is%3Aissue+label%3AEPIC) where all `EPIC` are listed. If there is a related `EPIC`, add the issue to the `EPIC` (by editing the description) and ping the product team.
+To do this, perform a search in the [Bug classification board](https://github.com/PrestaShop/PrestaShop/projects/11) where all EPIC are listed. If there is a related EPIC, add the issue to the EPIC (by editing the description) and ping the product team.
 
-If no `EPIC` clearly corresponds to the issue (or in case of doubt), just add the bug issue to the 1st column of the [kanban](https://github.com/PrestaShop/PrestaShop/projects/6) called "PrestaShop Next Major". The product team will regularly review the issues in this kanban and create new `EPIC`s when needed.
+If no Epic clearly corresponds to the issue (or in case of doubt), just add the bug issue to the 1st column of the kanban called "waiting for classification". The product team will regularly review the issues in this kanban and create new EPICs when needed.
```

Comment:
```suggestion
If no Epic clearly corresponds to the issue (or in case of doubt), just add the bug issue to the 1st column of the kanban called "waiting for classification". The product team will regularly review the issues in this kanban and create new Epics when needed.
```